package.json

```diff
@@ -61,6 +61,7 @@
     "into-stream": "^6.0.0",
     "jest": "^26.6.3",
     "jest-circus": "^26.6.3",
+    "mockdate": "^3.0.2",
```

Comment:
This can be removed since you are not using this package if I understood correctly.